opensrp-anc/src/main/java/org/smartregister/anc/library/adapter/ContactTasksDisplayAdapter.java

```diff
@@ -0,0 +1,62 @@
+package org.smartregister.anc.library.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.vijay.jsonwizard.constants.JsonFormConstants;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.smartregister.anc.library.R;
+import org.smartregister.anc.library.model.Task;
+import org.smartregister.anc.library.viewholder.ContactTasksViewHolder;
+
+import java.util.List;
+
+import timber.log.Timber;
+
+public class ContactTasksDisplayAdapter extends RecyclerView.Adapter<ContactTasksViewHolder> {
+    private List<Task> taskList;
+    private LayoutInflater inflater;
+    private Context context;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'context' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)